### PR TITLE
html: number validator

### PR DESCRIPTION
**Pull request by slavidichkov**

### CODE REVIEW

File: src/com/clouway/task7/numberValidator.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="validator.css">
+    <title>validator</title>
+</head>
+<body>
+Number: <input type="text" id="number" name="number">
+<p id="incorrect"></p><br>
+<button type="button" VALUE="isValidNumber" OnClick=isValidNumber('number')>is valid</button>
+<script src="validateNumber.js"></script>
+</body>
```
Comment (tsony-tsonev) [inline src/com/clouway/task7/numberValidator.html]: you should always format your html

File: src/com/clouway/task7/validateNumber.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+ */
+function isValidNumber(inputID) {
+    var validator = new RegExp("^(0)$|^([1-9]){1,10}$|^([1-9]){1,10}\.([0-9]){1,5}$|^(0)\.([0-9]){1,5}.$");
+    var inputValue = document.getElementById("number").value;
+    if (!validator.test(inputValue)) {
+        document.getElementById("incorrect").setAttribute("class", "invalid");
+        document.getElementById("incorrect").innerHTML = "number is incorrect";
+    } else {
+        document.getElementById("incorrect").setAttribute("class", "valid");
+        document.getElementById("incorrect").innerHTML = "number is correct";
```
Comment (tsony-tsonev) [inline src/com/clouway/task7/validateNumber.js]: you have duplication you could extract the get element by id in javascript function

**Comment by tsony-tsonev** [comment]
the comments are small you can merge this after fixing them

File: src/com/clouway/task7/validateNumber.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+/**
+ * Created by com on 16-2-26.
+ */
+function isValidNumber(inputID) {
+    var validator = new RegExp("^(0)$|^([1-9]){1,10}$|^([1-9]){1,10}\.([0-9]){1,5}$|^(0)\.([0-9]){1,5}.$");
+    var inputValue = document.getElementById("number").value;
+    if (!validator.test(inputValue)) {
+        document.getElementById("incorrect").setAttribute("class", "invalid");
```
Comment (tsony-tsonev) [inline src/com/clouway/task7/validateNumber.js]: use element.style. then css property with value cause now you override the whole css styles

Comment (tsony-tsonev) [inline src/com/clouway/task7/validateNumber.js]: or if you want to add another class just append it to the existing ones

File: src/com/clouway/task7/numberValidator.html (hunk 1 above)
Comment (slavidichkov) [inline src/com/clouway/task7/numberValidator.html]: fixed

File: src/com/clouway/task7/validateNumber.js (hunk 2 above)
Comment (slavidichkov) [inline src/com/clouway/task7/validateNumber.js]: fixed

File: src/com/clouway/task7/validateNumber.js (hunk 3 above)
Comment (slavidichkov) [inline src/com/clouway/task7/validateNumber.js]: I used  element.style.visibility= ....

Comment (slavidichkov) [inline src/com/clouway/task7/validateNumber.js]: fixed

**Pull request merged**